src/o365/teams/commands/report/teams-report-deviceusagedistributionusercounts.ts

```diff
@@ -68,6 +121,10 @@ class TeamsReportDeviceUsageDistributionUserCountsCommand extends GraphCommand {
         return `${args.options.period} is not a valid period type. The supported values are D7|D30|D90|D180`;
       }
 
+      if (args.options.outputFile && !fs.existsSync(path.dirname(args.options.outputFile))) {
+        return 'Specified outputFile path where to save the file does not exist';
```

Comment:
Should the output file exist upfront?

Reply:
No, it is not required. We are just checking the directory.

Reply:
Changed the parameter name from outputFile to outputFilePath to avoid confusion. Hope this works. Please let me know if you need any changes.